Objects/genericaliasobject.c

```diff
@@ -646,11 +646,11 @@ PyTypeObject Py_GenericAliasType = {
 PyObject *
 Py_GenericAlias(PyObject *origin, PyObject *args)
 {
-    gaobject *alias = PyObject_GC_New(gaobject, &Py_GenericAliasType);
+    gaobject *alias = (gaobject*) PyType_GenericAlloc(
```

Comment:
`PyType_GenericAlloc()` calls `_PyObject_GC_Malloc()` instead of `PyObject_GC_New()`.

Reply:
`_PyObject_GC_New` calls `_PyObject_GC_Malloc` + `_PyObject_Init`, which `PyType_GenericAlloc` also calls. What I am missing?

Reply:
All right then.

Please fix also `ga_new()`.

Reply:
``ga_new`` doesn't seem to need the fix. It uses ``tp_alloc`` which is set to ``PyType_GenericAlloc ``.